server/src/tests/ws.rs

```diff
@@ -819,7 +819,13 @@ async fn drop_client_with_pending_calls_works() {
 	init_logger();
 
 	let (handle, addr) = {
-		let server = ServerBuilder::default().build("127.0.0.1:0").with_default_timeout().await.unwrap().unwrap();
+		let server = ServerBuilder::default()
+			.ping_interval(std::time::Duration::from_secs(60 * 60))
```

Comment:
Is the point here that we send some requests that will never be responded to, and then expect that the client will never get a response back but that the server will shutdown fully?

I guess the big ping time prevents pings from helping move the server along? (is there a reason for 60 * 60 secs specifically; would any large enough time work?)

Might be worth a comment in any case here so future us knows why the piong interval was changed!

Reply:
yupp, if the graceful shutdown doesn't resolve in 60 seconds the timeout below will fail the test

Reply:
Gotcha,. thanks! I think it'd be worth making this explicit via comments because "with_default_timeout" below etc doesn't make it super obvious :)

Reply:
added a constant for that